papers/casper-basics/casper_basics.tex

```diff
@@ -68,7 +68,7 @@ \section{Introduction}
 
 There have been two major schools of thought in proof of stake algorithm design. The earlier of the two, \textit{chain-based proof of stake}, tries to closely mirror the mechanics of proof of work, featuring a chain of blocks and an algorithm that ``simulates'' mining by pseudorandomly assigning the right to create new blocks to stakeholders. This includes Peercoin\cite{king2012ppcoin}, Blackcoin\cite{vasin2014blackcoin} and Iddo Bentov's work\cite{bentov2016pos}.
 
-The other school, \textit{BFT-based proof of stake}, is based on a thirty year old body of research into \textit{Byzantine fault tolerant consensus algorithms} such as PBFT \cite{castro1999practical}. BFT algorithms tend to have strong and rigorously proven mathematical properties; for example, one can usually mathematically prove that as long as at more than $\frac{2}{3}$ of participants in the protocol are following the protocol correctly, then the algorithm cannot possibly finalize conflicting block hashes at the same time (``safety''); this result holds regardless of network latency. The suggestion of repurposing BFT algorithms for proof of stake was first introduced by Tendermint\cite{kwon2014tendermint}.
+The other school, \textit{BFT-based proof of stake}, is based on a thirty year old body of research into \textit{Byzantine fault tolerant consensus algorithms} such as PBFT \cite{castro1999practical}. BFT algorithms tend to have strong and rigorously proven mathematical properties; for example, one can usually mathematically prove that as long as at least $\frac{2}{3}$ of participants in the protocol are following the protocol correctly, then the algorithm cannot possibly finalize conflicting block hashes at the same time (``safety''); this result holds regardless of network latency. The suggestion of repurposing BFT algorithms for proof of stake was first introduced by Tendermint\cite{kwon2014tendermint}.
```

Comment:
Changed: "as long as at more than" --> "as long as at least"

Reply:
Believe this is incorrect. Should be `more than`. See https://github.com/ethereum/research/pull/23

Reply:
I'm pretty sure it is "at least". If at least 2/3 prepares, then checkpoint is justified. If justified, checkpoint can be finalized. If at least 2/3 commits, then checkpoint is finalized.

If two conflicting finalized blocks, then at least 1/3 of validators violated a slashing condition.

Reply:
Nope. Needs to be *more than* 2/3. 2/3 itself is not enough, and that's the fundamental result of BFT - 2 out of 3 is not sufficient to tolerate one byzantine fault. You need at least 4 nodes to tolerate one failure. So its specifically "more than 2/3", not "at least 2/3". 

On the contrary, "at least 1/3" is sufficient to violate safety. Just 1/3 is enough.

Hope that helps.

Reply:
@ebuchman Ah, you're right! I was reading this bit incorrectly. I appreciate the clarification.

I'll make this PR just in reference to the "dynasty" typo. Cheers